app/workers/generate_statistics_data_worker.rb

```diff
@@ -17,6 +17,10 @@ def perform(block_id)
     end
 
     CellOutput.upsert_all(cell_outputs_attributes) if cell_outputs_attributes.present?
-
+    block.ckb_transactions.find_each do |c|
+      res = CkbSync::Api.instance.directly_single_call_rpc(method: :get_transaction, params: [c.tx_hash, '0x0'])
+      c.bytes = (res['result']['transaction'].size - 2)/2
```

Comment:
should use transaction serialized data size directly, no need to call rpc

Reply:
use `Transaction#serialized_size_in_block` ?

Reply:
As you see, this is a background worker for update such information later.
And the former programmer also send request to RPC and allocate new Block object here:

https://github.com/nervosnetwork/ckb-explorer/blob/develop/app/workers/generate_statistics_data_worker.rb#L8

Reply:
> use Transaction#serialized_size_in_block ?
Yes,

> And the former programmer also send request to RPC and allocate new Block object here:


I think this rpc call can be omitted also by constructing from the hashmap:
```
node_block = CKB::Types::Block::from_h(block.to_raw)
```

Reply:
@ShiningRay will we refactor data fetching in this PR, or open another issue for it.